includes/class-alma-wc-plugin.php

```diff
@@ -79,40 +48,26 @@ class Alma_WC_Plugin {
 	/**
 	 * __construct
 	 *
-	 * @param string $file File.
-	 * @param string $version Version.
-	 *
 	 * @return void
+	 * @noinspection PhpIncludeInspection
 	 */
-	public function __construct( $file, $version ) {
-		$this->file    = $file;
-		$this->version = $version;
-
-		// Path.
-		$this->plugin_path   = trailingslashit( plugin_dir_path( $this->file ) );
-		$this->plugin_url    = trailingslashit( plugin_dir_url( $this->file ) );
-		$this->includes_path = $this->plugin_path . trailingslashit( 'includes' );
-
-		require_once $this->plugin_path . 'vendor/autoload.php';
-		require_once $this->includes_path . 'class-alma-wc-logger.php';
+	public function __construct() {
+		require_once ALMA_WC_PLUGIN_BASENAME . 'includes/alma-wc-functions.php';
+		require_once ALMA_WC_PLUGIN_BASENAME . 'vendor/autoload.php';
+		require_once ALMA_WC_PLUGIN_BASENAME . 'includes/class-alma-wc-autoloader.php';
```

Comment:
Why do we have to require it once again here, since it's been required in the plugin's main file before and this class here is constructed from the main file?

Reply:
oups ... you're all right.

I will pull-up all this `require` into the main file ;) (better cleaner stronger :) )